mediation_service/tests/performance/locustfile.py

```diff
@@ -0,0 +1,35 @@
+import os
+from common.auth import Auth
+from locust import HttpUser, task, between, run_single_user
+
+
+class LoadTestUser(HttpUser):
+    cookie = ""
+    wait_time = between(1, 5)
+    host = "https://internal-dev.api.service.nhs.uk/gp-connect-access-record-pr-94/AllergyIntolerance"
```

Comment:
@ChewieCB Can you parametrise the path here. It's pointing to your PR. `SERVICE_BASE_PATH` is the env var that we use in other places.

Reply:
Ah, nice catch - I've been using the Locust web UI which overrides this value so missed this debug url completely.

Reply:
79622c3848f217752008ac724c05a49b3f43881b